templates/element/variables_panel.php

```diff
@@ -24,6 +24,10 @@
     printf('<p class="warning">%s</p>', $error);
 endif;
 
+if (isset($varsMaxDepth)) {
+    printf('<p class="info">%s</p>', sprintf(__d('debug_kit', 'This view does not show very deep associations! Currently %s levels are being shown. You can overwrite this via the config key %s'), $varsMaxDepth, '<strong>DebugKit.variablesPanelMaxDepth</strong>'));
```

Comment:
The message is a bit tool long. It can just state the current max depth (no reference of associations is necessary) and how to configure it along with a warning that higher depth can lead to running out of memory.

Reply:
The 120 character limit per line is killing me here 😂

Reply:
I think the phrase `nested arrays` would work.